### DISCUSSION TITLE
Re-link web resource feature

### DISCUSSION THREAD

**Discussion by thanhkhoi2609** [Power-Maverick/DataverseDevTools-VSCode#67] [Ideas]
Thank you very much for the amazing tool.
However, I don't know how to re-link webresource at local machine with CRM server because I link with the wrong webresource and I want to re-link again.
What do you think about this feature?
In addition, I found that: if I delete the webresource in CRM then I choose upload and publish web resource, there is no new web resource is created with same name. I got message upload successfully but I can't find new web resource in CRM server and I can't link local web resource with new one.

Thank you very much,
Khoi

**Comment by Power-Maverick** [comment]
Sorry for the delayed response. All links are captured in `dvdt.linker.xml` file; which you can manipulate easily. But yes will also look into adding a re-link feature. 

Request you to please submit it as a feature under "Issues". Thank you.
